cms/templates/cms/home_page.html

```diff
@@ -46,9 +45,6 @@
 
       <div class="banner-wrapper">
         <div class="banner-wrapper-content">
-          <h1>
-            MicroMasters: Learn now, apply later (or never)
-          </h1>
```

Comment:
The template has already been updated to display the static text we want. 

The requirements for #1090 was just to remove the field from the model, not change the template.